### DISCUSSION TITLE
How to change the "block" color ?

### DISCUSSION THREAD

**Discussion by kevinadhiguna** [flameshot-org/flameshot#2115] [Q&A] [answered]
Hi, I want to change the "block" color from red to another color.

![image](https://user-images.githubusercontent.com/43397636/143378393-77cc7da5-d723-4598-b4cb-b2b915600dae.png)

I tried to modify the configuration but I didn't see any : 

![image](https://user-images.githubusercontent.com/43397636/143378487-70bf4342-d6a2-40d3-96fb-a2cf232b73e4.png)

How should I do that ? Any suggestion is appreciated.

OS: Ubuntu 20.04

**Comment by mmahmoudian** [comment, marked as answer]
There are are two easy ways. Before drawing the item:
1. right click anywhere on the screen to open the color wheel
2. Press space to open the sidebar and change the color from the color selector.

In the most recent versions of Flameshot you can also change the color of items after drawing them. You should first select the item and then use one the two methods I mentioned above.

**Reply by kevinadhiguna** [reply]
Worked like a charm, much appreciated !
